services/web/server/src/simcore_service_webserver/garbage_collector_utils.py

```diff
@@ -187,7 +187,7 @@ def log_context(log: Callable, message: str):
 
         yield
 
-        log("%s [DONE w/ SUCCESS]", message)
+        log("%s [DONE]", message)
```

Comment:
there is a ```log_context``` in service lib. Can we use it and remove that one instead? that would be really nice removing these duplicates on the way.

Reply:
I'm not sure why this was not included in the existing `log_context` from servicelib. There is a comment from @pcrespov maybe he can provide more context.